physicsnemo/models/diffusion_unets/_utils.py

```diff
@@ -17,56 +17,6 @@
 
 from typing import Any
 
-import numpy as np
-import torch
-
-
-def weight_init(shape: tuple, mode: str, fan_in: int, fan_out: int):
-    """
-    Unified routine for initializing weights and biases.
-    This function provides a unified interface for various weight initialization
-    strategies like Xavier (Glorot) and Kaiming (He) initializations.
-
```

Comment:
I may have missed it...did this function get moved somewhere else? Or is it unused?

Reply:
Nvm, I see it's moved to `nn` now